app/src/main/java/com/xmartlabs/doapp/controller/UserController.java

```diff
@@ -0,0 +1,47 @@
+package com.xmartlabs.doapp.controller;
+
+import android.support.annotation.NonNull;
+
+import com.raizlabs.android.dbflow.sql.language.SQLite;
+
+import com.xmartlabs.doapp.model.User;
+import com.xmartlabs.doapp.model.User_Table;
+
+import rx.Single;
+import rx.android.schedulers.AndroidSchedulers;
+import rx.schedulers.Schedulers;
+
+/**
+ * Created by scasas on 3/9/17.
+ */
+public class UserController extends Controller {
+  @NonNull
+  private final static UserController INSTANCE = new UserController();
+
+  public static UserController getInstance() {
```

Comment:
Use dagger to inject this dependency elsewhere instead of using a static variable